packages/terra-form-select/src/_FrameUtil.js

```diff
@@ -100,7 +100,7 @@ class FrameUtil {
     const { hasSearchChanged, searchValue } = state;
 
     if ((variant === Variants.TAG || variant === Variants.COMBOBOX)
-        && (variant !== Variants.TAG || searchValue.trim().length > 0)
```

Comment:
We should reach out to @neilpfeiffer to get his thoughts on this change. This line of code was previously allowing a user to clear the combobox by removing all characters within it. With this change, when a user clears the input field and tabs out, the previous value is reverted.

Previous: ( Clears on tab )
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/6720991/58712137-9da16580-8385-11e9-9d7f-c2d59f4a8fc7.gif)

With this change: ( Reverts back to green on tab )
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/6720991/58712152-a3974680-8385-11e9-8d77-d0b4d0558d3f.gif)


Steps:

1. Click on a combobox to open the dropdown
2. Select an option
3. Click back into the input field and backspace all character
4. Tab out of the combobox

Reply:
The difference here was a true empty string `''` was never added back into the combobox when it was cleared. What we are wanting to avoid in this PR is allowing a series of spaces to be added. `"  "`

Reply:
I spoke offline with @neilpfeiffer. We want to keep the previous functionality of allowing the user to clear the combobox by removing the characters, but we still want to prevent blank spaces from being added for the combobox as options.

Reply:
Thanks for following it up, @StephenEsser and @neilpfeiffer. In that case, could we simply do something like shown below? 

```diff
if ((variant === Variants.TAG || variant === Variants.COMBOBOX)
-  && (variant !== Variants.TAG || searchValue.trim().length > 0)
+  && (searchValue === '' || searchValue.trim().length > 0) // Allow empty string OR legit text.
   && hasSearchChanged
   && onSelect
   && (variant !== Variants.TAG || !FrameUtil.includes(props, searchValue))
   && (variant !== Variants.COMBOBOX || value !== searchValue)) {
      return true;
}
```

Reply:
Something similar, we want to make sure the Tag variant doesn't allow empty text to be created as a tag. 

The tricky part is I think we may need to treat blank strings as empty strings. Otherwise the workflow above happens again. 

A user makes a selection ( Green for example ), clicks back on the input, deletes all the characters, enters several spaces, presses tab. This would revert the blank spaces back to Green. I don't think we want that behavior.

Reply:
It may be helpful to document the behavior we intend for the combobox and the tag variants for this scenario clearly so @nramamurth has something actionable to code towards.
e.g. what steps should we test, what edge-cases should we consider.

Reply:
@bjankord 

That'd be great! In the meanwhile, I am trying to understand how all components within `terra-form-select` come together. I have left a [comment](https://github.com/cerner/terra-core/issues/2414#issuecomment-506196791) on the original issue to take up the discussion further there.

Reply:
I left a comment on the issue thread: https://github.com/cerner/terra-core/issues/2414#issuecomment-507322746